networkx/algorithms/flow/maxflow.py

```diff
@@ -13,13 +13,7 @@
 # Define the default flow function for computing maximum flow.
 default_flow_func = preflow_push
 # Functions that don't support cutoff for minimum cut computations.
-flow_funcs = [
```

Comment:
This is *technically* a publicly exposed variable that users could be accessing via `nx.flow.maxflow.flow_funcs`. However it definitely seems like this was never the intention and removing it seems unlikely that it will affect user code too much, so I'm not necessarily opposed to removing it without a warning cycle - any other thoughts?

Reply:
It would be nice to have a single source of truth for what the flow functions are and what cutoffs they support, and even preferred default, but this would take some significant refactoring of the various functions that are wrappers around flow functions. That said, leaving this variable in place is a step in the right direction, and it might even make sense for it to be a dict indicating cutoff support `flow_funcs_cutoff_support = {"preflow_push": False, ...}`

Reply:
I'm not so sure that one source of truth works as well as it should. This issue is entirely due to to trying to centralize information. But the presence of that information was not updated -- and nobody knew to look for it. If we're going to make a centralized dict containing all possible flow functions, let's do that and advertise it as such in a way that ensures it is current and accurate. Putting a dict in `connectivity` is unlikely to work in that way. Also, we only have one flow function that doesn't support cutoff, so testing against that one makes sense to me.